src/Lykke.Service.PayInternal.Services/Domain/BtcTransferRequest.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System;
+using System.Collections.Generic;
+using Lykke.Service.PayInternal.Core.Domain.Transfer;
+
+namespace Lykke.Service.PayInternal.Services.Domain
+{
+    public class BtcTransferRequest : ITransferRequest
```

Comment:
Didn't get the naming idea. It is named like request but is used as response

Reply:
Pls, see Core Comment-1.